ndcube/utils/wcs.py

```diff
@@ -427,9 +427,13 @@ def array_indices_for_world_objects(wcs, axes=None):
     else:
         world_indices = np.arange(wcs.world_n_dim)
 
-    object_names = np.array([wao_comp[0] for wao_comp in wcs.world_axis_object_components])
+    try:
```

Comment:
You don't need to do this. NDCube.wcs always is always a high level WCS. So you can always do .low_level_wcs

Reply:
No harm though, right?  Especially since the user has to manually enter the WCS.  This way the function supports High and Low level WCSs.

Reply:
I don't think we need to support both, all the rest of the code is assuming high, and the check just feels like it complicates the code.